package.json

```diff
@@ -73,6 +74,8 @@
     "@commitlint/cli": "^19.3.0",
     "@commitlint/config-conventional": "^19.2.2",
     "@next/eslint-plugin-next": "^14.2.3",
+    "@playwright/test": "^1.45.0",
```

Comment:
would they need to be in dependencies or devDependencies is fine too?

Reply:
It's dev dependency because we don't need in the bundle

Reply:
oh right yea I thought it was inside dependencies, all good then